vnext/package.json

```diff
@@ -65,23 +65,23 @@
     "@types/node": "^14.14.22",
     "@types/react": "^17.0.21",
     "@types/react-native": "^0.66.0",
-    "eslint": "7.12.0",
-    "eslint-plugin-prettier": "2.6.2",
-    "flow-bin": "^0.162.0",
+    "eslint": "^7.32.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "flow-bin": "^0.164.0",
     "jscodeshift": "^0.11.0",
     "just-scripts": "^1.3.3",
     "metro-config": "^0.66.0",
-    "prettier": "1.19.1",
+    "prettier": "^2.4.1",
```

Comment:
This is less cataclysmic than I thought the prettier upgrade would go 😂.

Are there any conflicts you ran into?

Reply:
The main weird issue is what you commented on below. I'm having to fork a couple of files because upstream those files are missing some prettier updates and are thus not matching what we have.

The other conflict right now is that two of the files in @office/iss flow-typed directory are not updating their formatting to the new prettier version. The prettier version for has been bumped and these files I believe are generated from yarn build, so I'm not sure what's missing here.